app/javascript/mastodon/features/compose/components/search.js

```diff
@@ -24,7 +24,7 @@ class SearchPopout extends React.PureComponent {
       <div style={{ ...style, position: 'absolute', width: 285, zIndex: 2 }}>
         <Motion defaultStyle={{ opacity: 0, scaleX: 0.85, scaleY: 0.75 }} style={{ opacity: spring(1, { damping: 35, stiffness: 400 }), scaleX: spring(1, { damping: 35, stiffness: 400 }), scaleY: spring(1, { damping: 35, stiffness: 400 }) }}>
           {({ opacity, scaleX, scaleY }) => (
-            <div className='search-popout' style={{ opacity: opacity, transform: `scale(${scaleX}, ${scaleY})` }}>
+            <div className='search-popout' style={{ opacity: opacity, transform: `scale(${scaleX}, ${scaleY})`, position: 'fixed', width: 257, top: 58 }}>
```

Comment:
While this appears to work fine in single-column mode, at least on Firefox, I am not comfortable with hardcoded values here, and this is not positioned correctly in the “advanced web interface” (multi-column). I am also not sure why you would hardcode the width.

A slightly cleaner approach might be to do it purely in CSS, and special-case the single-column interface, but even then I'm not too sure a hardcoded `top` value is the way to go.

Reply:
Fair enough, I hadn't even considered that it would be different in the other view. I've taken another approach instead of directly setting the CSS, which is attaching the overlay element to the search box. Previously, it was positioned as a detached top-level element.